src/Login/Login.tsx

```diff
@@ -132,6 +134,26 @@ export class Login extends React.Component<ILoginProps, { formData: any, sponsor
             </div>
           }
         </div>}
+        {this.props.authPage == "defaultPage" &&
+          <div className="top-form">
+            <AuthForm
+              formData={this.state.formData}
+              schema={Object.assign({}, this.props.schemas.validateEmail.schema, isStanfordEmail && { properties: { email: this.props.schemas.validateEmail.schema.properties.email }, required: ['email'] })}
```

Comment:
I think we can just simplify this to `this.props.schemas.validateEmail.schema`? (and same with uiSchema)